docs/functions.rst

```diff
@@ -4,12 +4,18 @@ Functions
 
 Flycs lets you define BigQuery functions. These functions will be created in BigQuery during the CI/CD pipeline and usable in your transformations.
 
-Example definition of a function in YAML:
+.. note::
+   Javascript functions using external code libraries are currently not supported.
+
+   If you still want to use javascript UDF importing external packages in your Flycs
+   transformation, you can create them in the raw dataset & reference it from this dataset.
```

Comment:
I don't get this. Why the raw dataset ? And I also do not want to have special behavior *when PII module is deactivated* in the documentation. We should have a solution that works for the normal setup of Flycs.

Reply:
```suggestion
   Javascript functions using external code libraries are not supported.
```

Reply:
Doc file changed to only inform JS external feature is not supported. (https://github.com/devoteamgcloud/flycs_sdk/pull/38/commits/0f8180e935ca9d179d2ce1a788e136181aca9068)